### DISCUSSION TITLE
Laravel MS-Graph Mail

### DISCUSSION THREAD

**Discussion by geisi** [HassanZahirnia/laravel-package-ocean#162] [Package Suggestions]
Laravel Mail driver for Microsoft Office 365 using the MSGraph API

[InnoGE/laravel-msgraph-mail](https://github.com/InnoGE/laravel-msgraph-mail)

The Office 365 SMTP Service is deprecated, so this package is the future proof way to send mails with Office 365.

**Comment by HassanZahirnia** [comment]
Hey Tim! 👋 Thanks for the suggestions 😊 I've added both packages!
